api/src/main/java/org/apache/apex/api/users/ControlTuple.java

```diff
@@ -16,15 +16,15 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.apex.api;
+package org.apache.apex.api.users;
```

Comment:
Even if control tuples are defined in a different library (not even necessary in malhar) they may need to be defined in the same package. Control tuples are inter-operator objects and file input operator may emit them, while aggregate operators will need to consume them, so it may be better to have them outside of a concrete operator package. Additionally, we may want to introduce property setter control tuples, in which case, platform may need to handle those control tuples.